spec/rubocop/cop/performance/sample_spec.rb

```diff
@@ -65,6 +65,18 @@
     expect(cop.messages).to be_empty
   end
 
+  it 'does not register an offense when calling shuffle.first with a block' do
+    inspect_source(cop, '[1, 2, 3, 4].shuffle.first { |e| e.odd? }')
```

Comment:
This should be removed altogether (I’ll do that later today), as `#find` doesn’t actually register as an offense (I mixed them up, apologies). The issue is actually with `shuffle(random: foo).find(&:bar)` registering as an offence – I assumed it’s the `shuffle…find` chain that triggers RuboCop (and then mixed it up wth `shuffle…first`), but it’s actually `shuffle(random: foo)`.